libs/sdk-backend-tiger/src/backend/organization/settings.ts

```diff
@@ -0,0 +1,39 @@
+// (C) 2022 GoodData Corporation
+import { IOrganizationSettingsService } from "@gooddata/sdk-backend-spi";
+import { IWhiteLabeling } from "@gooddata/sdk-model";
+import { TigerAuthenticatedCallGuard } from "../../types";
+
+export class OrganizationSettingsService implements IOrganizationSettingsService {
+    constructor(public readonly authCall: TigerAuthenticatedCallGuard) {}
+
+    public async setWhiteLabeling(whiteLabeling: IWhiteLabeling): Promise<void> {
+        this.setSettings("whiteLabeling", whiteLabeling);
+    }
+
+    private async setSettings(id: string, content: any): Promise<void> {
+        // It is not possible to PUT specific setting if it does not exist already,
+        // therefore we first clear it and POST a new one
+        await this.clearSettings(id);
```

Comment:
This is scary. What if clear settings succeeds but the create after it fails? Should we try backing up the old value just in case? Or we are ok with this risk?

Reply:
This is how theming do it. Maybe safer would be to fetch the value, if it exists update it and push it back, otherwise  create a new one.

Reply:
Maybe it could be handled on backend as well. To do not care about create/update of given settings. (to get rid of clear during setSettings)

Reply:
As discussed with pepi I modified the implementation to try to fetch data of given setting and then based on its non/existence call `create` or `update`. This implementation can be changed to pure `update` once NAS-4291 is implemented.